node-swc/src/types.ts

```diff
@@ -492,7 +492,21 @@ export interface JscConfig {
 
   experimental?: {
     optimizeHygiene?: boolean,
-    keepImportAssertions?: boolean
+    keepImportAssertions?: boolean,
+    /**
+     * Specify the location where SWC stores its intermidiate cache files.
+     * Currently only transform plugin uses this. If not specified, SWC will
+     * create `.swc` directories.
+     */
+    cache_root: string;
```

Comment:
I think this should be `cacheRoot` instead?

Reply:
🤦 oh you're right